Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.mm

```diff
@@ -236,8 +236,11 @@
     creationContext.surfacePool = &WebCore::IOSurfacePool::sharedPool();
 
 #if ENABLE(RE_DYNAMIC_CONTENT_SCALING)
-    if (m_parameters.includeDisplayList == IncludeDisplayList::Yes)
+    if (m_parameters.includeDisplayList == IncludeDisplayList::Yes) {
+        if (auto* context = m_layer->context())
+            creationContext.dynamicContentScalingResourceCache = context->dynamicContentScalingResourceCache();
```

Comment:
Is it okay to create `DynamicContentScalingBifurcatedImageBuffer` with a null `dynamicContentScalingResourceCache`? Should not we have a fallback and create a non shared resource case in this case?
```
if (auto* context = m_layer->context())
    creationContext.dynamicContentScalingResourceCache = context->dynamicContentScalingResourceCache();
else
    creationContext.dynamicContentScalingResourceCache = DynamicContentScalingBifurcatedImageBuffer::createResourceCache();
```

Reply:
No, this is fine. There's minimal point in making a non-shared non-persistent cache.